### PR TITLE
Forward events from redis objects to the redis claim of that instance

### PR DESCRIPTION
**Pull request by TheBigLee**
We want to forward the events of the redis objects (eg. statefulset) to the claim so that the end-user can see the events in the claim namespace.

We leverage our controller's functionality by setting the `vshn.net/forward-events-to` to the relevant objects.

## Checklist

- [x] The PR has a meaningful title. It will be used to auto generate the
      changelog.
      The PR has a meaningful description that sums up the change. It will be
      linked in the changelog.
- [x] PR contains a single logical change (to build a better changelog).
- [x] Categorize the PR by adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [ ] Link this PR to related issues or PRs.

### CODE REVIEW

**Review by Kidswiss — COMMENTED** [review]
LGTM

**Review by Kidswiss — APPROVED** [review]

File: package/main.yaml [hunk 1]
```diff
@@ -3,11 +3,11 @@ applications:
   - crossplane
 parameters:
   pkg.appcat:
-    componentVersion: v2.6.0
+    componentVersion: feature/sts_error_prop
     image:
       registry: ghcr.io
       repository: vshn/appcat
-      tag: v4.40.0
+      tag: feature/sts_error_prop
```
Comment (zugao) [inline package/main.yaml:10]: Remember to update the tag

File: package/main.yaml [hunk 2]
```diff
@@ -3,11 +3,11 @@ applications:
   - crossplane
 parameters:
   pkg.appcat:
-    componentVersion: v2.6.0
+    componentVersion: feature/sts_error_prop
```
Comment (zugao) [inline package/main.yaml:6]: Remember to update the tag

**Review by zugao — APPROVED** [review]
LGTM but we might already create a generic jsonnet function that would enable event forwarding for any service.

File: package/main.yaml (hunk 2 above)
Comment (TheBigLee) [reply package/main.yaml]: fixed

File: package/main.yaml (hunk 1 above)
Comment (TheBigLee) [reply package/main.yaml]: fixed

**Comment by TheBigLee** [comment]
> LGTM but we might already create a generic jsonnet function that would enable event forwarding for any service.

No I wouldn't do that. As we will use FunctionIO for all future services (and will have to move redis to full functionIO as well as some point), it's better to create a generic function there when the time comes.

**Pull request merged**